core/AmB2BSession.cpp

```diff
@@ -801,8 +801,17 @@ int AmB2BSession::relaySip(const AmSipRequest& req)
       }
     }
 
-    DBG("relaying SIP request %s %s\n", req.method.c_str(), req.r_uri.c_str());
-    int err = dlg->sendRequest(req.method, &body, *hdrs, SIP_FLAGS_VERBATIM);
+    int max_forwards;
+    if (req.max_forwards > (int)AmConfig::MaxForwards) {
+      max_forwards = AmConfig::MaxForwards;
+    } else {
+      max_forwards =  req.max_forwards - 1;
+    }
+    DBG("relaying SIP request %s %s %d\n", req.method.c_str(),
```

Comment:
MaxForwards should be checked in `AmBasicSipDialog::sendRequest`
This would allow MaxForward enforcement to work everywhere (including in SBC module)